bigquery/indexer.py

```diff
@@ -203,8 +203,14 @@ def _docs_by_id_from_export(storage_client, bucket_name, export_obj_prefix,
     for row in _rows_from_export(storage_client, bucket_name,
                                  export_obj_prefix):
         participant_id = row[participant_id_column]
+        # Document id is participant id; don't need it as a field.
         del row[participant_id_column]
-        row = {'%s.%s' % (table_name, k): v for k, v in row.iteritems()}
+        for k in row.keys():
+            # A BigQuery FLOAT column can have Infinity. Elasticsearch float
+            # doesn't handle Infinity, so discard.
+            if row[k] != 'Infinity':
```

Comment:
Seems like BigQuery float can also be '-Infinity', so we should probably handle that too.

Reply:
done